FormalConjectures/Wikipedia/UnionClosed.lean

```diff
@@ -1,5 +1,5 @@
 /-
-Copyright 2024 The Formal Conjectures Authors.
+Copyright 2025 The Formal Conjectures Authors.
```

Comment:
For this, shouldn't we allow different years in the header? (I think this is a sign I should fix the copyright header linter!)